extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/ItagItem.java

```diff
@@ -66,28 +69,19 @@
             new ItagItem(247, VIDEO_ONLY, WEBM, "720p"),
             new ItagItem(248, VIDEO_ONLY, WEBM, "1080p"),
             new ItagItem(271, VIDEO_ONLY, WEBM, "1440p"),
-            // #272 is either 3840x2160 (e.g. RtoitU2A-3E) or 7680x4320 (sLprVF6d7Ug)
-            new ItagItem(272, VIDEO_ONLY, WEBM, "2160p"),
             new ItagItem(302, VIDEO_ONLY, WEBM, "720p60", 60),
             new ItagItem(303, VIDEO_ONLY, WEBM, "1080p60", 60),
             new ItagItem(308, VIDEO_ONLY, WEBM, "1440p60", 60),
             new ItagItem(313, VIDEO_ONLY, WEBM, "2160p"),
-            new ItagItem(315, VIDEO_ONLY, WEBM, "2160p60", 60)
+            new ItagItem(315, VIDEO_ONLY, WEBM, "2160p60", 60),
+            new ItagItem(272, VIDEO_ONLY, WEBM, "4320p60", 60)
     };
 
     /*//////////////////////////////////////////////////////////////////////////
     // Utils
     //////////////////////////////////////////////////////////////////////////*/
 
-    public static boolean isSupported(int itag) {
-        for (ItagItem item : ITAG_LIST) {
-            if (itag == item.id) {
-                return true;
-            }
-        }
-        return false;
-    }
-
+    @Deprecated
```

Comment:
A reason for deprecating this/what to use instead in JavaDoc would be great here 😄 

I also noticed that https://github.com/TeamNewPipe/NewPipeExtractor/blob/6db4bea8ca51ac40a8f831527b274ad50895070e/extractor/src/main/java/org/schabi/newpipe/extractor/utils/DashMpdParser.java#L150 still relies on that code, so maybe some fixing has to be done there

Reply:
See https://github.com/TeamNewPipe/NewPipeExtractor/pull/488#discussion_r688385684